src/js/components/Video/Video.js

```diff
@@ -313,6 +313,7 @@ const Video = forwardRef(
             id: 'video.pauseButton',
             messages,
           }),
+          disabled: !playing,
```

Comment:
This seems to conflict with the code in the next line that checks for `playing` and calls `play` if not.